x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/Authentication.java

```diff
@@ -176,6 +176,7 @@ public void toXContentFragment(XContentBuilder builder) throws IOException {
             builder.field(User.Fields.REALM_TYPE.getPreferredName(), getAuthenticatedBy().getType());
         }
         builder.endObject();
+        builder.field(User.Fields.AUTHENTICATION_TYPE.getPreferredName(), getAuthenticationType().name());
```

Comment:
Should this be 
```suggestion
        builder.field(User.Fields.AUTHENTICATION_TYPE.getPreferredName(), getAuthenticationType().name().toLowercase(Locale.ROOT));
```
?
I can't think of many places in the ES APIs where we use ALL_CAPS values for enums.

Examples:
- https://github.com/elastic/elasticsearch/blob/v7.9.0/server/src/main/java/org/elasticsearch/rest/action/cat/RestHealthAction.java#L90
- https://github.com/elastic/elasticsearch/blob/v7.9.0/server/src/main/java/org/elasticsearch/action/support/WriteRequest.java#L63-L87